src/lib/wicked/connections.test.js

```diff
@@ -71,7 +71,7 @@ describe('#createConnection', () => {
             name: 'br0',
             bridge: {
                 stp: true,
-                ports: ['eth0', 'eth1']
+                ports: [{ device: 'eth0' }, { device: 'eth1' }]
```

Comment:
I rather prefer using interface names, instead of an object with a single `device` key.

Reply:
Oh, I misunderstood this change. LGTM :+1: